python/src/aiconfig/editor/client/src/components/prompt/PromptName.tsx

```diff
@@ -23,8 +23,8 @@ export default memo(function PromptName({ promptId, name, onUpdate }: Props) {
   return (
     <TextInput
       value={nameInput}
+      label="Prompt Name"
```

Comment:
You want the labels in both gradio and local editor?

Reply:
i use the mantine class to force label display: "none" in local-editor.